changedetectionio/__init__.py

```diff
@@ -500,45 +543,14 @@ def edit_page(uuid):
             if form.fetch_backend.data == datastore.data['settings']['application']['fetch_backend']:
                 form.fetch_backend.data = None
 
-            update_obj = {'url': form.url.data.strip(),
```

Comment:
ah you moved this to `def update_watch`, thanks, yeah i feel like theres a better way but this is a good thing anyway +1